packages/components/checkbox/src/CheckboxInput.tsx

```diff
@@ -62,7 +63,7 @@ export const CheckboxInput = forwardRef<HTMLButtonElement, CheckboxInputProps>(
         {...others}
       >
         <CheckboxIndicator>
-          {checked === 'indeterminate' ? indeterminateIcon : icon}
+          <Icon size="sm">{checked === 'indeterminate' ? indeterminateIcon : icon}</Icon>
```

Comment:
Not 100% sure if it's ok to use icon size here 🤔 WDYT ?

Reply:
I think it's fine because checkbox have a unique size :)